javascript2.lexer/external/jflex-1.4.3-license.txt

```diff
@@ -0,0 +1,308 @@
+Name: JFlex
+Version: 1.4.3
+Description: The Fast Scanner Generator
+License: GPL
+Origin: http://jflex.de/
+Type: compile-time
```

Comment:
This is more a note to self:
We need to update `VerifyLibsAndLicenses` (lines 314-324), so that for compile-time only dependencies, the GPL family as a whole is ok.